app/models/track.rb

```diff
@@ -1,15 +1,19 @@
 class Track < ApplicationRecord
   extend FriendlyId
   extend Mandate::Memoize
+  include Track::BuildStatus
```

Comment:
I've struggled with how I wanted to access the build status. Via a command? Maybe, but we don't do that often.
I've opted for an approach similar to `User::Roles`, where a module is included.